test/adapters/level_zero/v2/command_list_cache_test.cpp

```diff
@@ -167,3 +162,97 @@ TEST_P(CommandListCacheTest, ImmediateCommandListsHaveProperAttributes) {
         }
     }
 }
+
+TEST_P(CommandListCacheTest, CommandListsAreReusedByQueues) {
+    static constexpr int NumQueuesPerType = 5;
```

Comment:
What's the benefit of using `static` with `constexpr` ?

Reply:
I just kind of use `static constexpr` automatically when defining a const. Here, it doesn't matter much but in general, `static constexpr` allows you to use it in compile-time expressions (as template parameter, etc.)